app/components/CoinListCell/CoinListCellRangeView/CoinListCellRangeView.js

```diff
@@ -12,73 +12,108 @@ import Svg, {
 } from 'react-native-svg'
 import Colors from '../../../shared/styles/Colors'
 import coinListCellRangeViewStyles from './CoinListCellRangeViewStyles'
+import coinMarketcapDataFetcher from '../../../networkers/lowHightDataFetcher'
 
 
-const s = coinListCellRangeViewStyles
+export default class CoinListCellRangeView extends Component {
+  state = {
+    symbol: this.props.item.symbol,
+    priceUSD: this.props.item.price_usd,
+    low: null,
+    high: null,
+    fetchedData: false,
+  }
```

Comment:
why are we giving this component state again?

Reply:
oh I see its because we're waiting for the data to load in...in that case shouldn't we just have the `low:` `high:` `fetchedData:` fields since the others are being passed in from props.